vdslib/src/vespa/vdslib/distribution/distribution.cpp

```diff
@@ -10,7 +10,6 @@
 #include <vespa/config/print/asciiconfigreader.hpp>
 #include <vespa/vespalib/util/exceptions.h>
 #include <vespa/config-stor-distribution.h>
-#include <algorithm>
```

Comment:
The include statement is needed to get definition of std::sort (cf. PR #21439)

Reply:
K